packages/devextreme/testing/testcafe/tests/scheduler/tooltipBehaviour/tooltipBehavior.ts

```diff
@@ -117,3 +119,81 @@ safeSizeTest('The tooltip should hide after manually scrolling in the browser',
     adaptivityEnabled,
   }));
 });
+
+safeSizeTest('Tooltip on mobile devices should have enough hight if there are async templates (React)', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  const scheduler = new Scheduler('#container');
+
+  await t
+    .click(scheduler.collectors.find('7').element);
```

Comment:
Let's add one more scenario with a second click on the appointment.

I mean the following test case
- Click on the appointment
- Close tooltip
- Click on the appointment again
- Take a screenshot

We can extend this test case instead of creating a new one
(hide tooltip, one more click, and one more screenshot)

**Why:**
Because of this case, we have this change:
```typescript
const listHeight = getOuterHeight(this._list.$element().find('.dx-scrollable-content'));
``` 
(And this PR failed the manual testing phase on this test case last time)